pymc/logprob/transforms.py

```diff
@@ -569,8 +585,53 @@ def measurable_sub_to_neg(fgraph, node):
     return [pt.add(minuend, pt.neg(subtrahend))]
 
 
+@node_rewriter([log1p, softplus, log1mexp, log2, log10])
+def measurable_special_log_to_log(fgraph, node):
+    """Convert log1p, log1mexp, softplus, log2, log10 of `MeasurableVariable`s to log form."""
+    [inp] = node.inputs
+
+    if isinstance(node.op.scalar_op, Log1p):
+        return [pt.log(1 + inp)]
+    if isinstance(node.op.scalar_op, Softplus):
+        return [pt.log(1 + pt.exp(inp))]
+    if isinstance(node.op.scalar_op, Log1mexp):
+        return [pt.log(1 - pt.exp(pt.neg(inp)))]
```

Comment:
Are you sure the input should be negated? I think not

```suggestion
        return [pt.log(1 - pt.exp(pt.neg(inp)))]
```

Reply:
> Are you sure the input should be negated? I think not

Unless I'm missing something, it appears to be negated in other documentation https://www.rdocumentation.org/packages/VGAM/versions/1.1-8/topics/log1mexp.

Reply:
We are not using that implementation. The `Op` is defined here: https://github.com/pymc-devs/pytensor/blob/30e08e23c1ab0995eb8c34cc3914a9917228f2d0/pytensor/scalar/math.py#L1277